src/modules/stores/authentication/authentication.service.ts

```diff
@@ -39,6 +39,8 @@ type GetUserInfoDTO = {
   }[];
 };
 
+export type MFAInfoDTO = { type: 'none' } | { type: 'email' } | { type: 'phone'; phoneNumber: string };
```

Comment:
Here, `phoneNumber` can be undefined for a specific case:

When the MFA is set on the first time sign-in, the number that the policy is sending is not valid, which means that it will return as undefined even though the MFA is of type phone.